### DISCUSSION TITLE
wagmi next.js js project

### DISCUSSION THREAD

**Discussion by Rickshanelorie** [wevm/wagmi#3446] [General]
is there's any wagmi next.js js project template because im struggling to setup createConfig and WagmiProvider plz someone help

**Comment by tmm** [comment]
You can use the [`create-wagmi`](https://wagmi.sh/cli/create-wagmi) CLI to scaffold a basic Next.js project.

**Reply by Rickshanelorie** [reply]
there's only typescript im looking for Javascript
